jpaboard/src/test/java/com/hyunji/jpaboard/domain/post/domain/user/domain/UserTest.java

```diff
@@ -0,0 +1,50 @@
+package com.hyunji.jpaboard.domain.post.domain.user.domain;
+
+import com.hyunji.jpaboard.domain.user.domain.User;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+@Slf4j
+class UserTest {
+
+    @Test
+    void user_생성_성공() {
+        String name = "user01";
+        int age = 100;
+        String hobby = "hobby";
+
+        User user = User.create(name, age, hobby);
+
+        assertThat(user.getId()).isNull();
+        assertThat(user.getCreatedAt()).isNull();
+        assertThat(user).isNotNull()
+                .extracting(User::getName, User::getAge)
+                .isEqualTo(List.of(name, age));
```

Comment:
assert 대상이 컬렉션도 아닌데 굳이 extracting 할 필요가 있을까 싶어요

```suggestion
        assertThat(user.getName).isEqualTo(name);
        assertThat(user.getAge).isEqualTo(age);
```